qualtran/bloqs/mod_arithmetic/mod_addition_test.py

```diff
@@ -90,3 +97,51 @@ def test_classical_action_mod_add(prime, bitsize):
     for x in valid_range:
         for y in valid_range:
             assert b.call_classically(x=x, y=y) == cb.call_classically(x=x, y=y)
+
+
+@pytest.mark.parametrize('control', range(2))
+@pytest.mark.parametrize(
+    ['prime', 'bitsize'],
+    [(p, bitsize) for p in [11, 13, 31] for bitsize in range(1 + p.bit_length(), 8)],
+)
+@pytest.mark.parametrize('dtype', [QUInt, QMontgomeryUInt])
+def test_classical_action_cmodadd(control, prime, dtype, bitsize):
```

Comment:
nice! can you mark this exhaustive test as `pytest.mark.slow` and craft a `_quick` version: pick specific prime,bitsize pairs; take a couple (seeded) random values for x and y in valid_range

Reply:
done